pkg/shell/indexes.js

```diff
@@ -19,12 +19,17 @@
 
 import $ from "jquery";
 import cockpit from "cockpit";
+import React from "react";
+import ReactDOM from "react-dom";
+
+import { CockpitNav, CockpitNavItem } from "./nav.jsx";
+import { cHosts } from "./hosts.jsx";
 
 import * as base_index from "./base_index";
 
 const _ = cockpit.gettext;
 
-var shell_embedded = window.location.pathname.indexOf(".html") !== -1;
+var shell_embedded = window.location.pathname.indexOf(".html") !== -1; // TODO what does this do
```

Comment:
I'm not sure either, but some ideas:

 * Check that `firefox examples/embed-cockpit/application.html` looks as expected. (It looks fine to me though -- we do want the nav for "full server").
 * Aside from the use case above I'm not aware of another scenario where we have a .html in the URL bar, so maybe that's obsolete.
 * git archaeology points to commit cd3c8c558a407da for introducing this.